src/main/java/lesson9Text/TextAnalyzer.java

```diff
@@ -0,0 +1,22 @@
+package lesson9Text;
+
+public class TextAnalyzer {
+
+    public static void analyzeText(String text) {
+        if (!text.equals("") && text != null) {
```

Comment:
W tym momencie jak podasz null to poleci błąd, ponieważ najpierw wywołujesz equals("), musisz obrócić kolejność sprawdzeń:
`text != null && !text.equals("") `

Reply:
poprawione!